### PR TITLE
Jw prob

### PR DESCRIPTION
**Pull request by jinwoongseok**

### CODE REVIEW

**Comment by hw0404** [comment]
good

**Comment by jinwoongseok** [comment]
> good

Please approval

File: jwseok/Longest_Substring_Without_Repeating_Characters.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from collections import Counter
+
+
+class Solution:
+    def lengthOfLongestSubstring(self, s: str) -> int:
```
Comment (hw0404) [inline jwseok/Longest_Substring_Without_Repeating_Characters.py:5]: good

**Review by hw0404 — APPROVED** [review]

File: jwseok/Longest_Substring_Without_Repeating_Characters.py (hunk 1 above)
Comment (jinwoongseok) [inline jwseok/Longest_Substring_Without_Repeating_Characters.py:5]: 어떤 부분이 굿이라는거죠?

**Pull request merged**